src/threading.c

```diff
@@ -453,19 +455,88 @@ JL_DLLEXPORT jl_gcframe_t **jl_adopt_thread(void)
     return &ct->gcstack;
 }
 
+static char * get_path_from_sym(void *sym)
```

Comment:
This seems like it should be in `dlload.c` next to `jl_find_dynamic_library_by_addr`
(maybe call it `jl_find_dynamic_library_path_by_addr`)

Reply:
Looks like we already have `jl_pathname_for_handle` actually

Reply:
That one is slightly different since it takes in a library handle

Reply:
Right, but this seems simpler:
```julia
void *handle = jl_find_dynamic_library_by_addr(...);
const char *filepath = jl_pathname_for_handle(handle);
```

or is there a reason to re-implement that via the fused operation?

Reply:
I guess that's fine as well.